core/modules/node/node.install

```diff
@@ -109,7 +109,6 @@ function node_schema() {
       'node_frontpage'      => array('promote', 'status', 'sticky', 'created'),
       'node_status_type'    => array('status', 'type', 'nid'),
       'node_title_type'     => array('title', array('type', 4)),
-      'node_type'           => array(array('type', 4)),
```

Comment:
This index should still be left in here, as the column in the node table for "type" will still exist.